src/hostnet/hosts.mli

```diff
@@ -11,12 +11,12 @@ module Make(Files: Sig.FILES): sig
 
   type watch
 
-  val watch: ?path:string -> unit -> (watch, [ `Msg of string ]) result
+  val watch: ?path:string -> unit -> (watch, [ `Msg of string ]) result Lwt.t
```

Comment:
Not quite clear what the Lwt.t here does. Does it block until the watching begins, or does it block indefinitely until the watch terminates?

Reply:
It blocks until the watching begins. I should clarify that in a doc, as it's not very obvious.